src/data/faq.json

```diff
@@ -874,7 +874,7 @@
                                 "active": false,
                                 "textblock": [
                                     "In general, the digital proof of vaccination (or digital proof of test or recovery, respectively) via the Corona-Warn-App is a completely voluntary and complementary offer. If you do not have a digital certificate of vaccination or have lost it, the proof of vaccination via the well-known 'yellow booklet' (yellow vaccination passport) is usually still possible and valid. You can also prove Corona vaccinations with the printed Corona vaccination certificate for full vaccination. Digital certificates can be issued retrospectively by pharmacies.",
-                                    "<b>Important:</b> Please check the local regulations of the federal states."
+                                    "<b>Important:</b> Please check the local regulations of the federal states. Berlin, for example, requires that the proof of vaccination against or recovery from SARS-CoV2 must be digitally verifiable (see <a href='https://www.berlin.de/corona/massnahmen/verordnung/' target='_blank' rel='noopener noreferrer'>https://www.berlin.de/corona/massnahmen/verordnung/</a> §&nbsp;8a paragraph&nbsp;2 number&nbsp;4). This means that you may have to present a corresponding EU digital certificate for verification either in the Corona-Warn-App, the CovPass-App or as a paper printout."

```

Comment:
This is an incorrect reversion of a previous PR. It should not be in this PR at all.